catalyst/contrib/models/cv/segmentation/encoder/resnet.py

```diff
@@ -43,19 +43,20 @@ def __init__(
         state_dict: Union[dict, str, Path] = None,
     ):
         """
-        Specifies an encoder for segmentation network
+        Specifies an encoders for segmentation network
```

Comment:
```suggestion
        Specifies encoders for segmentation network
```

That's grammatically correct.